server/src/main/java/org/opensearch/search/aggregations/metrics/SumAggregationBuilder.java

```diff
@@ -62,15 +64,20 @@ public class SumAggregationBuilder extends ValuesSourceAggregationBuilder.LeafOn
         MetricAggregatorSupplier.class
     );
 
+    public static final ParseField METHOD_FIELD = new ParseField("method");
+
     public static final ObjectParser<SumAggregationBuilder, String> PARSER = ObjectParser.fromBuilder(NAME, SumAggregationBuilder::new);
     static {
         ValuesSourceAggregationBuilder.declareFields(PARSER, true, true, false);
+        PARSER.declareString(SumAggregationBuilder::method, SumAggregationBuilder.METHOD_FIELD);
     }
 
     public static void registerAggregators(ValuesSourceRegistry.Builder builder) {
         SumAggregatorFactory.registerAggregators(builder);
     }
 
+    private String method = null;
```

Comment:
The default should be `kahan`, right?

```suggestion
    private String method = "kahan";
```

Reply:
default changed in https://github.com/opensearch-project/OpenSearch/pull/6984/commits/7d50f500d045f1c8acb3d71318ef777cef6e6143